### DISCUSSION TITLE
Checkbox default color

### DISCUSSION THREAD

**Discussion by ediazm** [themesberg/flowbite-svelte#1504] [Ideas]
The color inside the checkbox is sometimes difficult to distinguish if it is disabled or not, depending on the background color that users are using (https://flowbite-svelte.com/docs/components/table#Checkbox). 

Having a prop boxColor: white | grey | orange

Would simplify not having to override it.

**Comment by shinokada** [comment]
If it's possible you can try https://svelte-5-ui-lib.codewithshin.com/forms/checkbox
